packages/docs-app/src/components/navHeader.tsx

```diff
@@ -17,7 +17,7 @@
 import type { NpmPackageInfo } from "@documentalist/client";
 import * as React from "react";
 
-import { Classes, HotkeysTarget2, type Intent, Menu, MenuItem, NavbarHeading, Popover, Tag } from "@blueprintjs/core";
+import { Classes, HotkeysTarget2, type Intent, Menu, MenuItem, NavbarHeading, Popover, PopupKind, Tag } from "@blueprintjs/core";
```

Comment:
Breaking this import out into multiple lines should satisfy prettier.

Reply:
Not needed after removing popupkind